doc-src/SASS_CHANGELOG.md

```diff
@@ -23,6 +23,13 @@
   function now accepts a `$bracketed` parameter that controls whether the
   returned list has brackets.
 
+* Passing a string to `call($function-name, $args...)` indicating which function to invoke is
+  now deprecated. Instead pass callable returned from `function-reference($function-name)`.
```

Comment:
As I was writing tests with `function(foo)` I just didn't like the way it looked. It felt confusingly close to a declaration of a function. I'm very open to alternate naming here.